jina/serve/runtimes/gateway/grpc/gateway.py

```diff
@@ -18,26 +19,30 @@ class GRPCGateway(BaseGateway):
 
     def __init__(
         self,
+        args: 'argparse.Namespace',
         port: Optional[int] = None,
         grpc_server_options: Optional[dict] = None,
         ssl_keyfile: Optional[str] = None,
         ssl_certfile: Optional[str] = None,
         **kwargs,
     ):
         """Initialize the gateway
+        :param args: runtime args
         :param port: The port of the Gateway, which the client should connect to.
         :param grpc_server_options: Dictionary of kwargs arguments that will be passed to the grpc server as options when starting the server, example : {'grpc.max_send_message_length': -1}
         :param ssl_keyfile: the path to the key file
         :param ssl_certfile: the path to the certificate file
         :param kwargs: keyword args
         """
-        super().__init__(**kwargs)
+        super().__init__(args=args, **kwargs)
+        self._setup_instrumentation()
         self.port = port
         self.grpc_server_options = grpc_server_options
         self.ssl_keyfile = ssl_keyfile
         self.ssl_certfile = ssl_certfile
         self.server = grpc.aio.server(
-            options=_get_grpc_server_options(self.grpc_server_options)
+            options=_get_grpc_server_options(self.grpc_server_options),
+            interceptors=[self.aio_tracing_server_interceptor()],
```

Comment:
Since I'm asking to remove args from constructor, if this needs a specific parameter,that parameter should be added to the constructor and then passed in this method call